subset/switches/src/main/java/switchtest/Main.java

```diff
@@ -2,31 +2,28 @@
 
 public class Main {
 
-	  public static void main(String[] args) {
-
-	    String ipAddress = "192.168.1.2";
-	    int interfacePort = 10;
-
-	    char[] command = new char[2];
-	    int count = 0;
-
-	    try {
-	      if (args != null) {
-	        if (args[0] != null) {
-	          ipAddress = args[0];
-	        }
-
-	        if (args[1] != null) {
-	          interfacePort = Integer.parseInt(args[1]);
-	        }
-	      }
-	    } catch (Exception e) {
-	      System.err.println("Exception main parse args:" + e.getMessage());
-	    }
-
-	    SwitchInterrogator switchInterrogator = new SwitchInterrogator(ipAddress, interfacePort);
-
-	    Thread switchInterrogatorThread = new Thread(switchInterrogator);
-	    switchInterrogatorThread.start();
-	  }
-	}
+  public static void main(String[] args) {
+
+    String ipAddress = "192.168.1.2";
```

Comment:
Both these constants seem like they'd do more harm that good. Probably better for the code to require the arguments rather than fill in arbitrary defaults.

Reply:
This is useful whilst developing within IDE.

Overwritten by args as soon as .jar is ran from command line

Reply:
The IDE has places for command-line settings, so if that's the use case then it's better to specify them there (since it's closer to actual intended program behavior).

Reply:
There's a place in the IDE for settings like this, so if that's the use case they should be sitting there. Then you also don't need to modify the code when running them. In IntelliJ there's a "command line" options field for a runtime profile.

Reply:
The IDE should have a place for this, and the settings should go there so you don't need to change the file at all when testing. In IntelliJ it's just a field called "command line" for the run profile.

Reply:
BTW I can't tell if GitHub is actually accepting these commands, so apologize if you receive 20 copies of this same one.

There's a place in the IDE for this setting, which is better because then you don't have to change the code at all to test. In IntelliJ it's called "command line options"